client/src/main/java/io/split/client/events/EventsSender.java

```diff
@@ -4,6 +4,7 @@
 import io.split.client.dtos.Event;
 import io.split.client.utils.Utils;
 import io.split.service.HttpPostImp;
+import io.split.service.SplitHttpClient;
 import io.split.telemetry.domain.enums.HttpParamsWrapper;
 import io.split.telemetry.storage.TelemetryRuntimeProducer;
 import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
```

Comment:
Please, remove this import that we aren't using anymore.

Reply:
will do